pii_recognition/utils.py

```diff
@@ -57,11 +57,16 @@ def dump_to_json_file(obj: Any, path: str):
 
 
 def stringify_keys(data: Dict) -> Dict[str, Any]:
```

Comment:
this function seems like a hack to me. why is this even necessary? why are the keys not already strings?

Reply:
The reason is that we did label grouping. When dumping to JSON we would have keys like `frozenset("BIRTHDAY", "DATE")` which cannot serialise directly. That is the main reason I've got this `stringfy_keys`.